Fix locale decimal mark

aleksandervines opened the pull request:
This should solve issue #22 by adding the option to specify comma as decimal separator in the "specify delimiters tab".

This also fixes a tiny bug where maxUploadSize was not set on the restore page.

It also fixes what I believe to be a small bug in calling validateUploadedTemplateFile, but I refer to issue #26 to resolve this.

There is still work to be done to add this to the decimal separator to the template file.

The two bugs are in smaller hotfix-branches as well, but I thought I should open this pull request to get the whole discussion going.

lesserwhirls commented on src/main/webapp/frontEndResources/js/stepFunctions.js:
```diff
@@ -185,7 +185,7 @@ function uploadRosettaTemplate(stepType, stepData) {
     } else if (stepType == "repopulateStep") {
         $(fileId).bind("change", function() {
             // Validate file being uploaded
-            var error = validateUploadedFile($("#file")[0].files[0], 1);
```
Same comment as below.

lesserwhirls commented:
@aleksandervines - this looks great. I see you also picked off the maxUploadSize from Unidata/rosetta#6 for the restore workflow. Thank you!

In terms of getting the decimal separator into the template file, I think it should show up as long as the key value pair is in the session storage.

The only other comment I would make is that if you wouldn't mind doing a git rebase to squash some of the merge messages before we finalize the PR, that'd be greatly appreciated!

aleksandervines commented:
Rebased. I initially wanted to show that the bugfixes were on different branches, so that you could select only those if you so wished, but it just got messy with all the merges.

Yes, I forgot I had fixed that small thing in #6 

The separator shows up in the quicksave file, which just dumps all the current session info, the template file is generated differently. And the template is missing a lot more than just this decimal separator. Again I refer to #26

aleksandervines commented:
@lesserwhirls 
I've further fixed the commits, also fixed some indent inconsistencies. I assume you want 4 spaces.

lesserwhirls commented:
Thanks @aleksandervines!
